### ISSUE TITLE
Update website and readme with logo

### ISSUE THREAD

**Issue by thymikee** [callstack/linaria#118]
We need it, sooner or later. Already asked my designer-brother (@pierzchala) to work on it in his free time (may be in couple of weeks).

The logo should resemble concepts like `flower`, `CSS`, `JS`.

**Comment by pierzchala** [comment]
Yo guys. Here it goes.
![callstack-linaria](https://user-images.githubusercontent.com/20886447/30990093-49b54eac-a4a0-11e7-8eef-3f66384de508.png)

**Comment by pierzchala** [comment]
Only free fonts used, no licence needed.

The font used for logo is Montserrat Alternates Regular [https://fonts.google.com/specimen/Montserrat+Alternates](url)

A matching font for text is Montserrat [https://fonts.google.com/specimen/Montserrat](url).
Basically, most sans-serif fonts will do. 

Have fun with it ;)

DOWNLOAD
[Callstack Linaria Logo.zip](https://github.com/callstack/linaria/files/1342351/Callstack.Linaria.Logo.zip)

**Comment by zamotany** [comment]
Awesome! 👍

**Comment by satya164** [comment]
Are you kidding me? This is freaking amazing! Thank you ❤️

**Comment by zamotany** [comment]
We need to update the website and redesign it cause I prefer this mockup @pierzchala did, it's amazing!

**Comment by morajabi** [comment]
I can give this a hand too :) It seems I'm obsessed with CSS-in-JS projects' websites 😝 (worked on the styled components website too)

**Comment by thymikee** [comment]
Any help on this is greatly appreciated! :)

**Comment by morajabi** [comment]
What do you mean by "Any help"? It's either the website or nothing, am I missing something?

**Comment by thymikee** [comment]
The website is already done, it just needs a redesign. Since we're not publishing it automatically, you're good to change any part of it, e.g. fonts and hero section, if you don't have time to finish all of it. What I mean is that we can update the website incrementally, in small changes if needed.

**Comment by morajabi** [comment]
Cool! Let's discuss the details then!

**Comment by thymikee** [comment]
For now we know that we'd like our hero area to look like this:

<img width="462" alt="screen shot 2017-12-17 at 22 11 19" src="https://user-images.githubusercontent.com/5106466/34083859-445d7716-e377-11e7-8b1a-92dd5973ac7b.png">

Rest of the page can stay as is (plus adjusting fonts and colors). Speaking fonts, I'd go with these:
* headings: [Montserrat Alternates](https://fonts.google.com/specimen/Montserrat+Alternates) (normal or medium weight, but we'll see what's better)
* body: [Montserrat](https://fonts.google.com/specimen/Montserrat) normal
* code: no changes (Fira Mono, falling back to monospace)

**Comment by morajabi** [comment]
Awesome, will take the first step tomorrow

**Comment by morajabi** [comment]
@thymikee Don't we want to have the website in a separate repo? `linaria-website` maybe?

**Comment by thymikee** [comment]
I don't see the point for now, maybe in some distant future.

**Comment by morajabi** [comment]
@thymikee Fair enough, do we want the hero to be this big? I mean to take all the screen?

**Comment by satya164** [comment]
@morajabi thanks for looking into it

> Because we can manage the issues, PRs, and releases in a separate repo, it's cleaner.

Keeping in the same repo means we can test the website against master of the library. Moving to separate repo will make it harder.

Let's do something like following:

![image](https://user-images.githubusercontent.com/1174278/34098571-c8e7f8fe-e402-11e7-8502-63e01b3ddb36.png)

**Comment by morajabi** [comment]
OMG @satya164 I love this!

**Comment by Trancever** [comment]
Hello Callstack engineers. I would like to work on this task if you still need some help. There've been no activity from **morajabi** on this for 28 days so i think he is not working on it anymore.

**Comment by thymikee** [comment]
@Trancever awesome, feel free to open a new PR for that.
@morajabi hope you don't mind? 🙂

**Comment by Trancever** [comment]
@thymikee I have encountered a problem with running website dev server. I got an error while executing "yarn run start". Of course, first i installed dependencies with yarn. 
There is a screenshot with the problem.
![error-linaria](https://user-images.githubusercontent.com/18584155/34948033-42d11b86-fa0c-11e7-8671-d8d8a039b9a7.PNG)

**Comment by zamotany** [comment]
@Trancever Can you open a separate issue? I'm pretty sure it's happening only on Windows. On Mac everything is fine. I'll try to reproduce it on Win10 later.

In the meantime you can check out https://docusaurus.io/
Ideally, we would like to have a documentation done with it with custom home/landing page as shown in mockup.

**Comment by Trancever** [comment]
@zamotany Yeah, i'll open an issue for that. 

So you not only want to change home page but also whole setup for creating documentation, am i right? Then what should i do now, work on the home page or check docusaurus and maybe try to setup it?

**Comment by satya164** [comment]
I'd suggest to finish the homepage first before we set up docusaurus :)

Also, can you share your email? I'll invite you to our Slack.

**Comment by Trancever** [comment]
My email is dawidu@onet.pl, thanks Satya :)

**Comment by zamotany** [comment]
It doesn't matter which one you do first, they can be done independently. I suggested that because, we're blocked by #194.

**Comment by Trancever** [comment]
Ok, I’ll install Linux tommorow to be able to work on the website. If it is done, i can have a look on docusaurus as Satya suggested.

**Comment by zamotany** [comment]
You can also try [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
